internal/storage/disk/disk.go

```diff
@@ -103,6 +105,33 @@ func (s *Store) GetDependents(_ context.Context, ids ...namer.ModuleID) (map[nam
 	return s.idx.GetDependents(ids...)
 }
 
+func (s *Store) InspectPolicies(ctx context.Context, _ storage.ListPolicyIDsParams) (map[string]*responsev1.InspectPoliciesResponse_Inspection, error) {
```

Comment:
Why not just push this to the index so that all stores that make use of the index can just call that method?